app/views/manuals/index.html.erb

```diff
@@ -6,14 +6,11 @@
   )
 %>
 
-<% if show_step_nav? %>
-  <%= render "govuk_publishing_components/components/step_by_step_nav_header", {
-    title: current_step_nav.title,
-    path: current_step_nav.base_path
-  } %>
+<% if step_nav_helper.show_header? %>
```

Comment:
This will not work if step_nav_helper is nil

Reply:
I don't think it should be nil, as it'll instantiate a new one when called (see the application controller).  I've got tests for when links are/aren't present, so I think this should be fine?

Reply:
I don't think it should be nil, as it'll instantiate a new one when called (see the application controller).  I've got tests for when links are/aren't present, so I think this should be fine?

I'll rebase too, thanks

Reply:
Got it! Thanks :)